lib/app.js

```diff
@@ -19,19 +21,12 @@ selector.addEventListener('change', function (event) {
 
 // Get locale of the current page
 function getCurrentLocale () {
+  var regexp
+  var location = window.location
```

Comment:
This file doesn't doesn't otherwise have `location` defined because it gets it from the web page. Even if technically this line is redundant I think it's good to do this so that when you just look at this file you still know what's going on and don't have to search for where `location` comes from.

That being said—is `window.<varbiable>` an ok way to do it? It seems to work but not certain it doesn't ruin things. What do you think?